CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -76,14 +83,20 @@ public static string GetMsalCacheLocation()
 
         internal static bool MsalEnabled()
         {
-            return GetEnabledFromEnvironment(MsalEnabledEnvVar, defaultValue: false);
+            return GetEnabledFromEnvironment(MsalEnabledEnvVar, defaultValue: true);
```

Comment:
This cannot default to true unless and until it does not regress the experience for MSA users

Reply:
ah that was just from testing - good catch!